neard/src/cli.rs

```diff
@@ -756,6 +762,16 @@ fn make_env_filter(verbose: Option<&str>) -> Result<EnvFilter, BuildEnvFilterErr
     Ok(env_filter)
 }
 
+#[derive(clap::Parser)]
+pub(super) struct ValidateConfigCommand {}
+
+impl ValidateConfigCommand {
+    pub(super) fn run(&self, home_dir: &Path) {
+        let _ = nearcore::config::load_config(&home_dir, GenesisValidationMode::Full)
+            .unwrap_or_else(|e| panic!("Error loading config: {:#}", e));
```

Comment:
same thing with panics here. It's probably cleaner to have this function return an `anyhow::Result<()>`, and just add a question mark to the place where this is called above. Giving a stacktrace here when there's a validatoion error doesn't feel great since nothing unexpected is happening. We can just print out the error to stderr normally and exit w/ nonzero code